clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -864,19 +876,24 @@ describe("Fides-js TCF", () => {
             cy.get("#fides-modal-link").click();
             cy.getByTestId("consent-modal").within(() => {
               cy.get("button").contains("Opt out of all").click();
-              cy.get("@FidesUpdated").then(() => {
-                // eslint-disable-next-line @typescript-eslint/no-unused-expressions
-                expect(spyObject).to.be.called;
-                const spy = spyObject.getCalls();
-                const { args } = spy[0];
-                expect(args[0]).to.deep.equal({
-                  data_sales: true,
-                  tracking: false,
+              cy.get("@FidesUpdated")
+                .should("have.been.calledOnce")
+                .its("lastCall.args.0.detail.extraDetails.consentMethod")
+                .then((consentMethod) => {
+                  expect(consentMethod).to.eql(ConsentMethod.accept);
```

Comment:
Hmm, that's a bit odd? How is it accept here, and then reject a few lines later? Is this assertion working?

Reply:
hm yes you're right, this `cy.on("fail")` I was using was scoped incorrectly and it was covering up the fact that `@FidesUpdated` wasn't being called at all due to the way I was mocking the savePreferencesFn. All fixed!